pylibui/core.py

```diff
@@ -24,6 +27,11 @@ def start(self):
         """
         libui.uiMain()
 
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.stop()
+        self.close()
+        
+
```

Comment:
Can you remove this extra line? I would like to maintain a coherent and clean style in the codebase..